keyboards/dekunukem/duckypad/info.json

```diff
@@ -1,8 +1,14 @@
 {
     "keyboard_name": "duckyPad",
```

Comment:
The changes to this file should be reverted, as most of this information is pulled from the config.h file.  And `keymap` isn't used at all.

Reply:
Reverted, thanks